CHANGELOG-unreleased.md

```diff
@@ -20,6 +20,7 @@ the released changes.
 - Added radial velocity methods for binary models
 - Support for wideband data in `pint.bayesian` (no correlated noise).
 - Added `DMWaveX` model (Fourier representation of DM noise)
+- Simulate correlated noise using `pint.simulate` functions and the `zima` script.
```

Comment:
`pint.simulation`, not `pint.simulate` (I think).

Also, is zima needed?  Or should it be clear that this works within zima too, but can work fine just using `pint.simulation`?

Reply:
corrected.

Reply:
Corrected.